source/local/local.go

```diff
@@ -144,17 +152,48 @@ func (s *localSource) GetFeatures() *nfdv1alpha1.Features {
 	return s.features
 }
 
-func parseFeatures(lines [][]byte) map[string]string {
-	features := make(map[string]string)
+func parseComments(line string, fileName string) map[string]interface{} {
```

Comment:
Hmm,  I don't like the ambiguous `map[string]interface{}` here. How about doing smth like

```golang
type parsingOpts struct {
    ExpiryTime time.Time
}

func parseComment(line string, opts *ParsingOpts) error {
...
```

I.e. the parsing func would update an opts struct. Also return an error (can be then logged on the upper level) which makes it easier to (unit)test, too.